team_production_system/serializers/availability.py

```diff
@@ -79,4 +85,32 @@ def validate_end_time(self, value):
             )
         return value
 
-    # TODO: Add validation for start times
+
+# V2 API #
+# The mentor availability serializer
+class AvailabilitySerializerV2(serializers.ModelSerializer):
+
+    class Meta:
+        model = Availability
+        fields = ['pk', 'mentor', 'start_time', 'end_time', 'status']
```

Comment:
Nit-pick: each field should probably get it's own line to increase readability
```suggestion
        fields = [
            'pk',
            'mentor',
            'start_time',
            'end_time',
            'status'
        ]
```